include/tvm/relay/attrs/nn.h

```diff
@@ -38,7 +38,7 @@ struct Conv2DAttrs : public tvm::AttrsNode<Conv2DAttrs> {
   IndexExpr channels;
   Array<IndexExpr> kernel_size;
   std::string data_layout;
-  std::string weight_layout;
+  std::string kernel_layout;
```

Comment:
rename `weight_layout` to `kernel_layout` to make relay consistent with nnvm and mxnet

Reply:
👍